src/Begin.f90

```diff
@@ -124,54 +134,53 @@ Subroutine BEGIN(param_file,spatial_file)
     if (source) then
 !
 !  Place holder for point source input
-!
-    end if 
-!
-!     The headwaters index for each cell in this reach is given
-!     in the order the cells are read
-!
-!     Card Type 3. Cell indexing #, Node # Row # Column Lat Long RM
-!
-!     Variable ndelta read in here.  At present, number of elements
-!     is entered manually into the network file (UW_JRY_2011/03/15)
-!
-    read(90,'(5x,i5,5x,i5,8x,i5,6x,a8,6x,a10,7x,f10.0,f5.0)')  &
-              node,nrow,ncol,lat,long,rmile1,ndelta(ncell)

-!
-!    Set the number of segments of the default, if not specified
-!
-    if (ndelta(ncell).lt.1) ndelta(ncell)=n_default
-    if(first_cell) then
-      first_cell=.false.
-      head_cell(nr)=ncell
-      x_dist(nr,0)=miles_to_ft*rmile0
+!
+    end if 
+!
+!     The headwaters index for each cell in this reach is given
+!     in the order the cells are read
+!
+!     Card Type 3. Cell indexing #, Node # Row # Column Lat Long RM
+!
+!     Variable ndelta read in here.  At present, number of elements
+!     is entered manually into the network file (UW_JRY_2011/03/15)
+!
+    read(90,'(5x,i5,5x,i5,8x,i5,6x,a8,6x,a10,7x,f10.0,i5)')  &
+              node,nrow,ncol,lat,long,rmile1,ndelta(ncell)
+!
+!    Set the number of segments of the default, if not specified
+!
+    if (ndelta(ncell).lt.1) ndelta(ncell)=n_default
+    if(first_cell) then
+      first_cell=.false.
+      head_cell(nr)=ncell
+      x_dist(nr,0)=5280.*rmile0
```

Comment:
Is there a reason to not hardcode the conversion from feet to miles? I think defining "miles_to_ft" in one of the "Block" scripts is best.